package.json

```diff
@@ -1,5 +1,12 @@
 {
     "name": "bugsnag-js",
+	"description": "The Bugsnag is an error reporter for JavaScript that automatically detects and reports errors.",
```

Comment:
I would remove the word "The" when referring to Bugsnag. I think this should read.

> Automatically detect JavaScript errors on your websites, collect detailed diagnostic information, and get notified about errors in real-time.